IETF-RFC.md

```diff
@@ -151,6 +152,13 @@ Since the Invite Flow does not require either Party to type or remember the `use
 
 Also, a different `userID` could be given out to each contact, to avoid correlation of identities.
 
+If the inviteAcceptDialog uses a WAYF page, it MAY consist of a fixed list of servers, in addition to, or instead of, a free-text input where any OCM server can be entered. This is especially useful if the Invite Sender is part of a network of associated OCM Servers. The Invite Sender's server MAY use a central database to store the fixed list of OCM servers. In that case the central database MUST be exposed as a JSON document. 
```

Comment:
I'd replace `database` -> `directory`. Also, shall we add that: the Directory MUST offer an HTTP endpoint, such that a `HTTP GET` would return a JSON document with the following fields...